src/schemas/json/mtad.json

```diff
@@ -66,7 +66,7 @@
                         "pattern": "^[A-Za-z0-9_\\-\\.]+$"
                     },
                     "type": {
-                        "description": "The module type that defines the design-time tools for the module.",
+                        "description": "Defines a pre-defined set of configurations for the module deployment. Use module type 'custom' to have no predefined configurations and use module parameters to define such",
```

Comment:
I don't like the `Defines a pre-defined` part. Also, `custom` should soon be deprecated in favor of `application`, so we should probably advertise it instead.
What do you think about:
```
The name of the module type that would provide default configurations for this module's deployment. You can use the module type `application` to have no predefined configurations and then define everything in the module's parameters.
```